frontend/apps/web/components/jobs/SchemaTable/RowAlert.tsx

```diff
@@ -5,24 +5,21 @@ import {
   TooltipTrigger,
 } from '@/components/ui/tooltip';
 import { ExclamationTriangleIcon } from '@radix-ui/react-icons';
-import { Row } from '@tanstack/react-table';
 import { ReactElement } from 'react';
-import { SchemaConstraintHandler } from './schema-constraint-handler';
+import {
+  ColumnKey,
+  SchemaConstraintHandler,
+} from './schema-constraint-handler';
 
 interface Props {
-  row: Row<{ schema: string; table: string; column: string }>;
```

Comment:
TS was continually unhappy about this even though it builds. This update just makes it less type dependent on the Row type and pushes it to the caller.